server/routes/playlist.js

```diff
@@ -3,7 +3,7 @@ const jwt = require('../utils/jwt')
 const router = express.Router()
 
 const Playlist = require('../models/Playlists')
-const { BadRequestError } = require('../utils/errors')
+const SimilarityCount = require('../models/SimilarityCount')
```

Comment:
This dependency should likely go down now that you have a root rather than up(../)

Reply:
I only set up a root for the client side. I could not figure out how to set a root for the server side without using external libraries.